flexmeasures/api/v3_0/users.py

```diff
@@ -42,6 +42,24 @@
 account_schema = AccountSchema()
 
 
+class BaseKwargsSchema(Schema):
+    page = fields.Int(required=False, validate=validate.Range(min=1))
+    per_page = fields.Int(
+        required=False, validate=validate.Range(min=1), load_default=1
+    )
+    filter = SearchFilterField(required=False)
+    sort_by = fields.Str(
```

Comment:
Actually, the allowed values of this field are only relevant to the user query endpoint, and *not* for the audit log endpoint (see the comment below). So here we would have specific sub-classes of `PaginationSchema`, to actually set the allowed field options per paginated endpoint.

I found that the audit log endpoint only allows event_datetime (which is hard-coded in lines 530 and 611). This seems like an unnecessary restriction - let's extend that code to allow also `active_user_name`.